### ISSUE TITLE
Ladybird document.bgColor not working

### ISSUE THREAD

**Issue by PGHales** [SerenityOS/serenity#23929]
Hey, I'm a student at the University of Utah. My team and I are working on fixing bugs for the Ladybird browser as part of our final project and we found this one that seems like it would be pretty simple for us to fix.

We found the problem when visiting this [site](http://randomcolour.com/). Normally, it sets the background color of the web page to some random color. It does so by setting document.bgColor using Javascript. However, the background color does not get set. Our guess is that document.bgColor is not implemented (or if it is implemented, it isn't working). According to the specification at [https://developer.mozilla.org/en-US/docs/Web/API/Document/bgColor](https://developer.mozilla.org/en-US/docs/Web/API/Document/bgColor) this feature is deprecated, so it's not required to have, but something that would be nice to have for compatibility with sites that still use it.

We created a small website that isolates the issue:
```html
<body>
<script type="text/javascript">
document.bgColor="#00ff00";
</script>
```
Running this page in Ladybird doesn't change the background color to green, but it does do this in a normal web browser like Chrome or Firefox. In fact, both Chrome and Firefox appear to set the bgcolor attribute of the `<body>` tag when setting document.bgColor, so we will likely do something similar to that to fix the issue, especially since a page like:
```html
<body bgcolor="red">
``` 
does correctly set the background color in Ladybird. However, we aren't familiar with the code base yet, so any tips on where can look to start fixing the issue are greatly appreciated. Thanks!

**Comment by tcl3** [comment]
You're correct that we don't currently implement the `document.bgColor` property and that's why the page isn't working.

It looks like this attribute is mentioned in this part of the HTML specification: https://html.spec.whatwg.org/multipage/obsolete.html#dom-document-bgcolor.

This part of the specification is covered by these Web Platform Tests, which may also be worth a look:
* http://wpt.live/html/obsolete/requirements-for-implementations/other-elements-attributes-and-apis/document-color-01.html
* http://wpt.live/html/obsolete/requirements-for-implementations/other-elements-attributes-and-apis/document-color-02.html
* http://wpt.live/html/obsolete/requirements-for-implementations/other-elements-attributes-and-apis/document-color-03.html
* http://wpt.live/html/obsolete/requirements-for-implementations/other-elements-attributes-and-apis/document-color-04.html

**Comment by tcl3** [comment]
Fixed by #23956.
